integration_tests/mockApis/componentApi.ts

```diff
@@ -20,6 +20,13 @@ const componentsNoServices: HeaderFooterSharedData = {
       type: 'INST',
       caseloadFunction: 'GENERAL',
     },
+    {
+      caseLoadId: 'KMI',
+      currentlyActive: false,
+      description: 'HMP Hewell',
+      type: 'INST',
```

Comment:
`HMP Hewell` description appears in 2 x caseloads. Might be nicer to alter the description too, even if it doesn't affect the outcome of tests?